regression/lua/rt_gk_bgk_cross_relax_1x2v_p1.lua

```diff
@@ -0,0 +1,221 @@
+mu_max_ion = mass_ion * math.pow(5.0 * vti_par, 2.0) / (2.0 * B0) -- Domain boundary (ion velocity space: magnetic moment direction).
+poly_order = 1 -- Polynomial order.
+basis_type = "serendipity" -- Basis function set.
+time_stepper = "rk3" -- Time integrator.
+cfl_frac = 1.0 -- CFL coefficient.
+
+t_end = 0.1 / nu_ion -- Final simulation time.
+num_frames = 1 -- Number of output frames.
+dt_failure_tol = 1.0e-4 -- Minimum allowable fraction of initial time-step.
+num_failures_max = 20 -- Maximum allowable number of consecutive small time-steps.
+
+gyrokineticApp = Gyrokinetic.App.new {
+
+  tEnd = t_end,
+  nFrame = num_frames,
+  dtFailureTol = dt_failure_tol,
+  numFailuresMax = num_failures_max,
+  lower = { -0.5 * Lz },
+  upper = { 0.5 * Lz },
+  cells = { Nz },
+  cflFrac = cfl_frac,
+
+  --basis = basis_type,
+  polyOrder = poly_order,
+  timeStepper = time_stepper,
+
+  -- Decomposition for configuration space.
+  decompCuts = { 1 }, -- Cuts in each coodinate direction (x-direction only).
+
+  -- Boundary conditions for configuration space.
+  periodicDirs = { 1 }, -- Periodic directions (x-direction only).
+
+  geometry = {
+    geometryID = G0.Geometry.MapC2P,
+    world = { 0.0, 0.0, 0.0 },
+
+    -- Computational coordinates (x, y, z) from physical coordinates (X, Y, Z).
+    mapc2p = function (t, zc)
+      local xp = { }
+      
+      xp[1] = zc[1]
+      xp[2] = zc[2]
+      xp[3] = zc[3]
+
+      return xp[1], xp[2], xp[3]
+    end,
+
+    -- Magnetic field strength.
+    bmagFunc = function (t, zc)
+      return B0
+    end
+  },
+
+  -- Electrons.
+  elc = Gyrokinetic.Species.new {
+    charge = charge_elc, mass = mass_elc,
+    
+    -- Velocity space grid.
+    lower = { -vpar_max_elc, 0.0 },
+    upper = { vpar_max_elc, mu_max_elc },
+    cells = { Nvpar, Nmu },
+    polarizationDensity = n0,
+    noBy = true,
+
+    -- Initial conditions.
+    projection = {
+      projectionID = G0.Projection.BiMaxwellian,
+
+      densityInit = function (t, xn)
+        return n0 -- Electron total number density.
+      end,
+      parallelTemperatureInit = function (t, xn)
+        return T_par_elc -- Electron parallel temperature.
+      end,
+      perpendicularTemperatureInit = function (t, xn)
+        return T_perp_elc -- Electron perpendicular temperature.
+      end,
+      parallelVelocityInit = function (t, xn)
+        return upar_elc -- Electron parallel velocity.
+      end,
+
+      correctAllMoments = true
+    },
+
+    collisions = {
+      collisionID = G0.Collisions.BGK,
+
+      normalizeNu = true,
+      referenceDensity = n0,
+      referenceTemperature = Te,
+
+      selfNu = function (t, xn)
+        return nu_elc
+      end,
+
+      numCrossCollisions = 1,
+      collideWith = { "ion" },
+
+      correctAllMoments = true,
+      iterationEpsilon = 1.0e-12,
+      maxIterations = 10,
+      useLastConverged = true
+    },
```

Comment:
This isn't for this PR, but (FYI @JunoRavin ) I think this set of inputs for correction operations should be grouped in a single struct/table. That way we can pass those (optional) inputs as a group to various places where we do corrections (e.g. initialization, BGK, reactions, etc)